contracts/limitOrder/LimitOrderBook.sol

```diff
@@ -219,4 +228,66 @@ contract LimitOrderBook is
 
         return signer;
     }
+
+    //
+    // INTERNAL VIEW
+    //
+
+    function _formatDecimals(
+        uint256 price,
+        uint8 fromDecimals,
+        uint8 toDecimals
+    ) internal pure returns (uint256) {
+        // LOB_ID: Invalid Decimals
+        require(fromDecimals <= toDecimals, "LOB_ID");
+
+        return price.mul(10**(toDecimals.sub(fromDecimals)));
+    }
+
+    function _getPriceByRoundId(address baseToken, uint80 roundId) internal view returns (uint256) {
+        ChainlinkPriceFeed chainlinkPriceFeed = ChainlinkPriceFeed(IBaseToken(baseToken).getPriceFeed());
+        (uint256 price, ) = chainlinkPriceFeed.getRoundData(roundId);
+        return _formatDecimals(price, chainlinkPriceFeed.decimals(), 18);
+    }
+
+    function _verifyTriggerPrice(LimitOrder memory order, uint80 roundIdWhenTriggered) internal view {
```

Comment:
Wonder why this is internal but `verifySigner` is public? Are we sure that no one needs to use this on their own bots?

Reply:
Ah, backend service used to call `verifySigner()` to do an extra check when storing orders into DB, but it didn't do that now.

Can make both functions public or internal. I think public is ok, they're useful utility functions sometimes. What do you prefer?

Reply:
Decided to make both `verifySigner()` and `verifyTriggerPrice()` internal (so I don't have to write tests for them).